distribution.md

```diff
@@ -24,7 +24,7 @@ A buildpackage MUST exist as either an OCI image on an image registry, an OCI im
 
 A `.cnb` file MUST be an uncompressed tar archive containing an OCI image. Its file name SHOULD end in `.cnb`.
 
-Each FS layer blob in the buildpackage MUST contain a single buildpack at the following file path:
+Each buildpack-related FS layer blob in the buildpackage MUST contain a single buildpack at the following file path:
```

Comment:
Thanks for the suggestion @jkutner. I agree that `buildpack-related` may be a little too broad. At the same time mentioning containment on both sides of the statement feels a bit cyclical. What do you think about this?

```suggestion
Each FS layer blob in the buildpackage intended to be a buildpack MUST contain a single buildpack at the following file path:
```

Reply:
👍

Reply:
Great, adjusted!